.github/workflows/check-format.yml

```diff
@@ -27,5 +27,4 @@ jobs:
         pip install -e .[test]
     - name: Run code format checks
       run: |
-        black --check .
-        flake8 --enable-extensions=BCS src
+        tox -e linters_check
```

Comment:
What the difference between "linters" and "linters_check"?

Reply:
This is read only.`black` will take no action here